### PR TITLE
update grunt and other dev-dependencies

### PR DESCRIPTION
**Pull request by jelhan**
Tests were hanging at 
> Running "qunit:main" (qunit) task
> Testing tests/tests.html 
and never finish. Updating dependencies fixed it for me.

My configuration:
```
$ nodejs -v
v6.9.4
$ npm -v
3.10.10
$ grunt --version
grunt-cli v1.2.0
grunt v0.4.5
```

| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,24 +25,24 @@
...
+    "grunt-contrib-csslint": "^2.0.0",
+    "grunt-contrib-cssmin": "^1.0.2",
+    "grunt-contrib-jshint": "^1.1.0",
+    "grunt-contrib-less": "^1.4.0",
+    "grunt-contrib-qunit": "^1.2.0",
+    "grunt-contrib-uglify": "^2.0.0",
+    "grunt-jscs": "^3.0.1",
+    "grunt-string-replace": "^1.3.1",
+    "load-grunt-tasks": "^3.5.2",
+    "phantomjs-prebuilt": "~2.1.7",
```
Comment (vsn4ik) [inline package.json]: Latest is `2.1.14`.

Comment (jelhan) [inline package.json]: [Tilde range](https://docs.npmjs.com/misc/semver#tilde-ranges-123-12-1) matches all higher patch version as well. `~2.1.7` matches `2.1.x` if x is greater or equal 7.

Comment (acrobat) [inline package.json]: `~2.1.7` is ok for now. We can change it if we upgrade to a 2.x version

**Comment by acrobat** [comment]
Thanks @jelhan! Btw are you interested in becoming a collaborator of bootstrap-datepicker and help us get this 1.x version stable (and maybe in the future start work on 2.0)? You've opened a few PR's and looks like you know your way around js!

**Comment by jelhan** [comment]
@acrobat Thanks for the offer but I don't have much spare time these days.

**Comment by acrobat** [comment]
No problem! Thanks for the contributions!

**Pull request merged**